packages/mobile/android/app/src/main/res/values/styles.xml

```diff
@@ -2,7 +2,7 @@
   <style name="AppTheme" parent="Base.V0.AppTheme"/>
 
   <!--  Base theme, also see API specific overrides in values-vXX/styles.xml -->
-  <style name="Base.V0.AppTheme" parent="Theme.AppCompat.Light.NoActionBar">
+  <style name="Base.V0.AppTheme" parent="Theme.AppCompat.DayNight.NoActionBar">
```

Comment:
Why do we need to change this?

Reply:
i don't have a better answer than i tried to copy the changes here https://react-native-community.github.io/upgrade-helper/?from=0.63.4&to=0.66.4 - my naive thinking is that newer versions of android has light/dark themes but i didn't look into it much. do you prefer to keep it unchanged?

Reply:
Sounds good. Maybe we should double check how the app is impacted when the dark theme is selected (Settings -> Display -> Theme).

Reply:
I quickly checked this, the app looks the same as usual in dark mode (except for the native things like keyboard and alerts which are dark) 🙈